Packs/AzureSentinel/Scripts/MicrosoftSentinelSetOwnerButton/MicrosoftSentinelSetOwnerButton.yml

```diff
@@ -0,0 +1,30 @@
+args:
+- name: owner_email
+  required: true
+  description: The owner email to set as user principal name.
+commonfields:
+  id: MicrosoftSentinelSetOwnerButton
+  version: -1
+comment: This script is used to set owner in remote system.
+name: MicrosoftSentinelSetOwnerButton
+outputs:
+  - contextPath: AzureSentinel.Incident.Owner.assignedTo
```

Comment:
adjust output according to requested change above

Reply:
Simply put here the outputs that appear in the executed command?